path_rotate.go

```diff
@@ -70,7 +70,12 @@ func (b *backend) pathRotateCredentials() []*framework.Path {
 }
 
 func (b *backend) pathRotateRootCredentialsUpdate(ctx context.Context, req *logical.Request, _ *framework.FieldData) (*logical.Response, error) {
-	return nil, b.rotateRootCredential(ctx, req)
+	err := b.rotateRootCredential(ctx, req)
+	if err != nil {
+		b.Logger().Error("failed to rotate root credential:", "rotationID", req.Path, "cause", err.Error())
```

Comment:
Should we just say error instead of cause?

Reply:
i have no memory of writing 'cause'!

Yeah i'll change it to error.